src/pages/HouseRules.tsx

```diff
@@ -0,0 +1,296 @@
+import { useState, useRef } from 'react';
+import HouseRule from '../components/HouseRule';
+import schedule from '../assets/ruleSchedule.png';
+import LineDivider from '../components/LineDivider';
+
+interface RuleListProps {
+  rules: string[];
+  activeIndex: number | null;
+  onClick: (index: number) => void;
+}
+
+const HouseRules = () => {
+  const [activeIndex, setActiveIndex] = useState<number | null>(null);
+
+  const ruleRefs = useRef<(HTMLDivElement | null)[]>([]);
+
+  const toggleRule = (index: number) => {
+    setActiveIndex((prevIndex) => (prevIndex === index ? null : index));
+
+    if (ruleRefs.current[index]) {
+      ruleRefs.current[index].scrollIntoView({
+        behavior: 'smooth',
+      });
+
+      const currentRule = ruleRefs.current[index];
+
+      const title = currentRule.querySelector('h5');
+
+      if (title) {
+        title.style.color = '#ff6700';
+      }
+    }
+  };
+
+  const rules = [
+    'Algemene Verordening Gegevensbescherming (AVG)',
+    'Lidmaatschap & Contributie',
+    'Rookbeleid',
+    'Schenkbeleid',
+    'Banenbeleid',
+    'Vrijwilligersbeleid',
+    'Huisdierenbeleid',
+    'Klachtbehandeling',
+    'Slotbepalingen',
+  ];
+
+  const RuleList: React.FC<RuleListProps> = ({
+    rules,
+    activeIndex,
+    onClick,
+  }) => {
+    return (
+      <ol>
+        {rules.map((rule, index) => (
+          <li
+            key={index}
+            onClick={() => onClick(index)}
+            className={activeIndex === index ? 'active' : ''}
+          >
+            {rule}
+          </li>
+        ))}
+      </ol>
+    );
+  };
+
+  return (
+    <>
+      <article>
+        <section className="houserule-container">
+          <h1>
+            Het Huishoudelijk reglement van JBV De Walnoot bevat de hieronder
+            benoemde artikelen:
+          </h1>
+          <RuleList
+            rules={rules}
+            activeIndex={activeIndex}
+            onClick={toggleRule}
+          />
+        </section>
+        <LineDivider />
+        <div ref={(el) => (ruleRefs.current[0] = el)}>
+          <HouseRule
```

Comment:
```suggestion
          <HouseRule 
          ref={(el) => (ruleRefs.current[0] = el)}
```

i dont think it's needed to wrap this in a div you can just ref it to the component